AudioKitSynthOne.xcodeproj/project.pbxproj

```diff
@@ -1423,12 +1423,12 @@
 				TargetAttributes = {
 					941645C920B3597300D62851 = {
 						CreatedOnToolsVersion = 9.3.1;
-						DevelopmentTeam = 9W69ZP8S5F;
+						DevelopmentTeam = 69WU6LWMSQ;
```

Comment:
Also exclude changes to this project file from the PR since it's just your internal configuration.

Reply:
Same as Above